docs/smart-agent-quickstart.md

```diff
@@ -1,132 +1,298 @@
+## Step 5. Add a New Monitor 
 
-#### Step 2: Configuration
+You may also want to add and configure various monitors for your environment. 
 
-The default configuration file should be located at `/etc/signalfx/agent.yaml` on Linux
-and `\ProgramData\SignalFxAgent\agent.yaml` on Windows.
-Also, by default, the file containing your SignalFx API token should be located at
-`/etc/signalfx/token` on Linux and `\ProgramData\SignalFxAgent\token` on Windows.
+* To [add a new monitor](#Monitors).
+* For [Common configuration options](https://docs.signalfx.com/en/latest/integrations/agent/monitor-config.html).
+* For [Windows monitor configurations](https://docs.signalfx.com/en/latest/integrations/agent/windows.html).
+* For [Linux monitor configurations](https://docs.signalfx.com/en/latest/integrations/agent/monitor-config.html)
 
-In the referenced example agent.yaml configuration files below, the default
-location for the token file is used.
 
-- [Linux Default Configuration File](https://github.com/signalfx/signalfx-agent/blob/master/packaging/etc/agent.yaml)
+### Configure optional metrics for your monitors. 
 
-- [Windows Default Configuration File](https://github.com/signalfx/signalfx-agent/blob/master/packaging/win/agent.yaml)
+See individual monitor pages for your monitor from the lists in Step 5 or see the [Integrations tab](https://app.signalfx.com/#/integrations) in the SignalFx application for Monitor Services.
 
-##### Configure your endpoints
+## Step 6. Add a New Observer 
 
-By default, the Smart Agent will send data to the `us0` realm.
-If you are not in this realm, you will need to explicitly set the
-`signalFxRealm` option in your config like this:
+See [Observers](#Observers).
 
-```
-signalFxRealm: YOUR_SIGNALFX_REALM
-```
+## Step 7. Explore Dashboards to display and compare data from various sources. 
 
-To determine if you are in a different realm and need to
-explicitly set the endpoints, check your profile page in the SignalFx
-web application.
+See [Dashboards](https://docs.signalfx.com/en/latest/dashboards/index.html).
 
-If you want to explicitly set the ingest, API server, and trace endpoint URLs,
-you can set them individually like so:
+See [Built-In Dashboards and Charts](https://docs.signalfx.com/en/latest/getting-started/built-in-content/built-in-dashboards.html). 
 
-```
-ingestUrl: "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"
-apiUrl: "https://api.YOUR_SIGNALFX_REALM.signalfx.com"
-traceEndpointUrl: "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com/v1/trace"
-```
+See best practices for [Better Dashboards](https://docs.signalfx.com/en/latest/reference/best-practices/better-dashboards.html). 
 
-They will default to the endpoints for the realm configured in `signalFxRealm`
-if not set.
+## To learn more about Smart Agent, see the [15-Minute SingnalFx Quick Start](https://docs.signalfx.com/en/latest/getting-started/quick-start.html). 
```

Comment:
That's not really a "learn more about the SA" link. You could say "To learn more about how your data is presented in SignalFx, see (quickstart doc)." If you keep this, it should be the first suggestion. The suggestions should go from most general to to most specific.

Reply:
done.